t2.lic

```diff
@@ -122,7 +122,7 @@ before_dying do
     stop_script(script_name) if Script.running?(script_name)
   end
   # Don't bother releasing cyclic for NMU guilds
```

Comment:
Can you remove this comment, too?

Reply:
Done. Sorry, that was sloppy on my part.